priv/app/tracing_switch.jsx

```diff
@@ -16,24 +19,18 @@ export default class TracingSwitch extends React.Component {
   }
 
   handleClick(event) {
-    var spec = this.state.status === "running" ? "pause" : "all";
-
-    $.ajax({
-      url: "/api/trace_set",
```

Comment:
I think the button's main job should still be to stop tracing (via the "api/trace_set" API). It is an important functionality that shouldn't be removed. The user should be able to quickly stop all impact of XProf on a production system by just pushing this brake-button.
As a side-effect it should also stop updating the graphs (because there is no new data coming in anyway when tracing is stopped) Btw this functionality can be used as well to just pause the graphs for a bit.
So there would be no way to just pause the graph but meanwhile still going on with tracing, but this is a trade-off for the sake of simplicity (until someone actually asks for this)